### PR TITLE
[ANDWL-275] ixo add

### PR DESCRIPTION
**Pull request by kimheeje12**

### CODE REVIEW

File: app/src/main/java/wannabit/io/cosmostaion/base/BaseChain.java [hunk 1]
```diff
@@ -302,6 +306,7 @@ public static ArrayList<BaseChain> SUPPORT_CHAINS() {
         result.add(SIF_MAIN);
         result.add(STARGAZE_MAIN);
         result.add(IOV_MAIN);
+        result.add(IXO_MAIN);
```
Comment (Kwonhyukjoon) [inline app/src/main/java/wannabit/io/cosmostaion/base/BaseChain.java:309]: Check ordering

File: app/src/main/java/wannabit/io/cosmostaion/base/chains/ChainFactory.java [hunk 2]
```diff
@@ -157,6 +157,9 @@ public static ChainConfig getChain(BaseChain baseChain) {
                 case IOV_MAIN:
                     return new Starname();
 
+                case IXO_MAIN:
```
Comment (Kwonhyukjoon) [inline app/src/main/java/wannabit/io/cosmostaion/base/chains/ChainFactory.java:160]: Check ordering

**Comment by soaryong-c** [comment]
conflict

**Review by Kwonhyukjoon — APPROVED** [review]
👍

**Review by soaryong-c — APPROVED** [review]
👍

**Pull request merged**